src/components/curator-actions/BountyCardDetails.svelte

```diff
@@ -63,15 +65,16 @@
 							<p>{formatDate(bounty.expiryDate)}</p>
 						</section>
 					{/if}
-					{#if bounty.curator}
-						<div class="mt-4 lg:mt-0">
-							<p class="text-xs">Curator</p>
-							<CopyableAddress address={bounty.curator} />
-						</div>
-					{/if}
 				</div>
-				<!--  -->
 			</section>
+			{#if bounty.curator}
+				<button
+					class="flex justify-end lg:mr-12 2xl:mr-44 text-accent bg-white font-bold rounded-md lg:h-auto lg:pt-1 lg:max-w-32"
+					on:click={() => {
+						curatorsDialogOpen = true;
+					}}>SHOW CURATORS</button
+				>
```

Comment:
```suggestion
					on:click={() => {
						curatorsDialogOpen = true;
					}}>
  					SHOW CURATORS
					</button>
```
Would this look nicer in the code?